### PR TITLE
Direct Match using HTMX

### PR DESCRIPTION
**Pull request by ColeDCrawford**
This should get merged after #57 as it includes those changes, forgot to switch back to main and there are enough things there that I don't want to try to cherry-pick my way out of it. The changes that matter are in `data/views.py` and `es_search.js`. @rcavezza take a look - think this should cover the functionality you are going for in https://github.com/searchrx/searchrx/pull/56. It uses HTMX but in a bit of an unconventional way; normally that gets initialized by adding HTML properties to elements, but the elements we want to use are not available at pageload so we don't have control over them. I tried adding them after pageload, and that worked but then blocked the Searchkit POST for ES data, so I instead am triggering an `htmx.ajax()` call to get the new HTML.

Still need to handle clearing data.

@lucyzsunshine this will need some styling and maybe titles or labels for the data. The drug labels should look similar to the sections, but different enough someone can tell they are different kinds of data. The template that will need to be styled is `dle/dle/data/templates/data/_label_search_results.html`. Can wait to do that until this is in `main` or feel free to contribute on this branch if you get to it before then.

Screen Shot 2023-04-26 at 9 24 03 AM

### CODE REVIEW

**Comment by rcavezza** [comment]
Sweet - thanks! @ColeDCrawford When we get this merged in, I can make some tweaks to get this displayed a little differently.

File: dle/api/views.py [hunk 1]
```diff
@@ -17,6 +17,7 @@
 def searchkit(request: HttpRequest) -> JsonResponse:
     """Core search API which gets proxied to Elasticsearch"""
     es = get_client()
+    print(request.body)
```
Comment (rcavezza) [inline dle/api/views.py:20]: Probably an accidental debugging print statement here, but I can remove it when I make some tweaks to the visuals after this gets merged.

**Comment by rcavezza** [comment]
Also gotta remind everyone to do a fresh build for the updated requirements.txt after merged.

**Comment by ColeDCrawford** [comment]
Thanks Bobby. Cleaned this up and now it handles when data is cleared and when there is a query in the URL on pageload. If I can find an event which Searchkit fires once it's finished loading then we can get rid of all the logic in that `beforeSearch` hook, but all I'm using it for now is to fire a custom event to say "SK is ready" and then attach the event listeners from the script in `search.html`. Maybe move that script to a separate module in the future.

**Comment by lucyzsunshine** [comment]
WIP, will finish before the end of tonite

**Pull request merged**